src/server/schema/requirements.js

```diff
@@ -0,0 +1,40 @@
+const BaseSchema = require('./base');
+const mongoose = require('mongoose');
+
+module.exports = function(config) {
+  const RequirementSchema = new BaseSchema({
+    skills             : [{
+      type: String, lowercase: true
+    }],
+    description        : {
+      type: String, maxlength:250
+    },
+    details            : {
+      type:String, maxlength: 4096
+    },
+    min_experience     : {
+      type: Number
+    },
+    max_experience     : {
+      type: Number,
+    },
+    business_unit      : {
+      type: String, required: true, enum: config.types.business_type
+    },
+    manager            : {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: 'Interviewer' // TODO: define &  require Interviewer Schema
+    },
+    hr_contact         : {
+      type:mongoose.Schema.Types.ObjectId,
+      ref:'Interviewer' // TODO: define &  require Interviewer Schema
+    },
+    active             : {
```

Comment:
this field should be in base schema, not here. If we intend to use it, then it should be inherited by every schema.